### ISSUE TITLE
device-random and device-virtual breakage in snap

### ISSUE THREAD

**Issue by tonyespy** [edgexfoundry/device-sdk-go#451]
A recent change to device-sdk-go introduced new command-line changes which break the device services (-random, and -virtual) included by default in the snap. Both of these device service were being built from the fuji branch instead of from a source tag in the snap, which also contributed to this breakage. This commit updates the command-line arguments for both, and also sets git-tag: v1.1.1 to be used for both instead of the fuji branch.

# 🐞 Bug Report

### Affected Services
device-random and device-virtual in the snap

The issue is located in: snapcraft.yaml

### Is this a regression?

Partially, when the snap was rebuilt to pick up a previous fix for Ubuntu Core (see issue edgexfoundry/edgex-go#2356), the device service parts picked up the recent device-sdk-go change because the snap was building them from the fuji branch instead of a tag.

## 🔥 Exception or Error
If the device services are enabled, they will fail to start:

```
$ sudo snap set edgexfoundry device-random=on
$ snap services edgexfoundry | grep random
edgexfoundry.device-random    enabled  inactive  -
```

## 🌍  Your Environment
**Deployment Environment:**
snap

**EdgeX Version:**
fuji

**Comment by tonyespy** [comment]
I should have a PR for master shortly. And as with issue #2368 (snap ExecutorPath fix) I'll go ahead reference this issue in my PR for master.

**Comment by tonyespy** [comment]
This issue shouldn't have been moved to device-sdk-go. The main EdgeX snap is called "edgexfoundry" and it the associated snap packaging is part of the edgex-go repo. Originally the edgexfoundry snap included both device-random and device-virtual device services so that when someone install the snap, they had some of the developer focused device services already installed.

The bug was caused by the fact that version 1.1.x of device-sdk-go made a breaking change to how command-line arguments were handled, so both services were failing start when enabled in the snap. For fuji, the command lines for both were fixed. For Geneva, we pulled device-random from the snap as it's didn't really add much value to developers (and reduced the size of the snap), and fixed the command-line arguments for device-virtual.

This was addressed in fuji via: https://github.com/edgexfoundry/edgex-go/pull/2367
And addressed in geneva/master via: https://github.com/edgexfoundry/edgex-go/pull/2426
